src/applications/vaos/containers/DateTimeRequestPage.jsx

```diff
@@ -93,30 +59,45 @@ export class DateTimeRequestPage extends React.Component {
   };
 
   goForward = () => {
-    this.validate();
-    if (this.userSelectedSlot()) {
-      this.props.routeToNextAppointmentPage(this.props.router, pageKey);
-    } else {
+    const { data, router } = this.props;
+    const { calendarData } = data || {};
+    this.validate(calendarData);
```

Comment:
Could we get rid of this validate call? I think it would be triggered by data changes already.

Reply:
Just tested it out and looks like it is needed now that we are no longer using SchemaForm.  With SchemaForm, clicking the forward button triggered the `onChange` but without it, it doesn't.